app/templates/security.hbs

```diff
@@ -0,0 +1,52 @@
+<PageHeader @title='Security Information' />
+
+<TextContent @boxed={{true}}>
+
+  <h2 id='crates-io-security'>Security of crates.io itself</h2>
+
+  <p>Safety is one of the core principles of Rust, and to that end, we would like to ensure that cargo, crates.io, docs.rs, and
+    related tools have secure implementations. To disclose security vulnerabilities in
+    <a href='https://github.com/rust-lang'>any repository in the rust-lang organization</a>, please follow the
```

Comment:
```suggestion
    <a href='https://github.com/rust-lang'>any repository in the rust-lang organization</a> and the crates.io website (not the crates hosted on it), please follow the
```

Since this page is hosted on crates.io and linked in the footer, it might be good to explicitly state that vulnerabilities in crates.io should be reported through the project's security policy.

We all know in the project this is true, but putting myself in the shoes of someone interacting with crates.io the first time, having to go check whether crates.io is a repository in the rust-lang organization is annoying :slightly_smiling_face:

Reply:
I've attempted to clarify.